### ISSUE TITLE
Asciidoc asciidoc

### ISSUE THREAD

**Issue by amiroslaw** [scullyio/scully#1330]
### Description
When I build scully with asciidoc file I've got:
`When you want to use asciidoc, you need to install the optional asciidocor.js module by: npm install -s asciidoctor.js`
I suppose the error is from the asciidoc.ts in scully/libs/scully/src/lib/fileHanderPlugins/

#### 💻Your Environment

**Angular Version:** 10.0.0
**Scully Version:** 1.0.0
asciidoctor.js 1.5.9

**Comment by SanderElias** [comment]
@amiroslaw did you try installing `asciidoctor.js`?

**Comment by amiroslaw** [comment]
Yes of course I've tried with asciidoctor.js 1.5.9 and asciidoctor 2.2.4.

**Comment by SanderElias** [comment]
I was just double-checking. I will test this in an isolated project, and add it to my list. You might be the first one (after me) that actually tries this :-D 
iirc we do have a test that hits AD tho. I'll keep you posted.
